es6.js

```diff
@@ -0,0 +1,13 @@
+const printName = (name) => "Hii" + name;
+console.log(printName('Avinash'));
+const printBill = (name, bill) => {
+    return `Hi ${name}, please pay: ${bill}`;
+}
+console.log(printBill("avi",500));
+const person = {
+    name: "Noam Chomsky",
+    age: 92
+}
+let {name:n,age:a}=person;
+console.log(n);
```

Comment:
let { name, age } = person;
console.log(name);
console.log(age);

this is enough